tests/test_database.py

```diff
@@ -12,6 +13,25 @@
 import audformat.testing
 
 
+def create_misc_table(
```

Comment:
Should we add `audformat.testing.add_misc_table()` or update `audformat.testing.add_table()`?

I don't know if it really is useful, but the code we use in the tests looks like this at the moment:

```python
audformat.testing.add_table(other1, 'table', ...)
other1['misc'] = create_misc_table(...)
```
which automatically raises the question.

Reply:
On the other hand we also have `create_table()` inside `tests/test_table.py`

Reply:
> update audformat.testing.add_table()

Yes, I also thought about updating `testing.add_table()` or adding `testing.add_misc_table()` to support `MiscTable` objects. Should we do it in this PR?

The reason we have additionally have `create_table()` and `create_misc_table()` is to create a `Table` / `MiscTable` without a database object.

Reply:
Yes, I would also propose to not include this in the next release.
I created https://github.com/audeering/audformat/issues/260